Explorer/Assets/DCL/CharacterTriggerArea/Components/CharacterTriggerAreaComponent.cs

```diff
@@ -1,26 +1,26 @@
 using DCL.ECSComponents;
-using System;
+using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.Serialization;
 
 namespace DCL.CharacterTriggerArea.Components
 {
     public struct CharacterTriggerAreaComponent : IDirtyMarker
     {
-        public Action<Collider> OnEnteredTrigger;
-        public Action<Collider> OnExitedTrigger;
         public Vector3 AreaSize;
         public CharacterTriggerArea MonoBehaviour;
         public bool TargetOnlyMainPlayer;
+        public readonly HashSet<Transform> EnteredThisFrame;
+        public readonly HashSet<Transform> ExitedThisFrame;
```

Comment:
Same case: `feat/sdk-camera-mode-area_refactor-proposal-1`

Reply:
Applied your [refactor proposal branch suggestions](https://github.com/decentraland/unity-explorer/compare/feat/sdk-camera-mode-area_refactor...feat/sdk-camera-mode-area_refactor-proposal-1) and adapted the test coverage at [this commit](https://github.com/decentraland/unity-explorer/pull/340/commits/e99b338855fb32fb1dd8ef22e2dead5ed8ab8cef).